components/kserve/kserve_config_handler.go

```diff
@@ -25,18 +25,22 @@ const (
 func (k *Kserve) setupKserveConfig(ctx context.Context, cli client.Client, logger logr.Logger, dscispec *dsciv1.DSCInitializationSpec) error {
 	// as long as Kserve.Serving is not 'Removed', we will setup the dependencies
 
+	var defaultDeploymentMode DefaultDeploymentMode
```

Comment:
Since it selects mode from default to Serverless, the name may be misleading. Probably, it is `developmentMode` now? I do not remember details where the default mode can be changed, so ignore if I'm wrong.

Reply:
the field is called deploymentMode in Kserve so we are keeping the terminology consistent. The options for deploymentMode are RawDeployment or Serverless

Reply:
So, you confirmed that variable defaultDeploymentMode  should be called deploymentMode.